packages/mermaid/src/docs/syntax/zenuml.md

```diff
@@ -295,3 +298,17 @@ zenuml
       API->BookingService: rollback status
     }
 ```
+
+## Integrating with your library/website.
+
+Zenuml uses the experimental lazy loading & async rendering features which could change in the future.
+
+You can use this method to add mermaid including the zenuml diagram to a web page:
+
+```html
+<script type="module">
+  import mermaid from 'https://cdn.jsdelivr.net/npm/mermaid@10/dist/mermaid.esm.min.mjs';
+  import zenuml from 'https://cdn.jsdelivr.net/npm/@mermaid-js/mermaid-zenuml@00.1.0/dist/mermaid-zenuml.esm.min.mjs';
```

Comment:
Extra 0 in @00.1.0

Reply:
Fixed in https://github.com/mermaid-js/mermaid/pull/4405/commits/82eccb70faebc158f260b833c2a1dd4b349194ea